simulation_parameters/Constants.cs

```diff
@@ -740,6 +740,9 @@ public static class Constants
     // Cooldown for AI for toggling engulfing
     public const float AI_ENGULF_INTERVAL = 300;
 
+    // Probability, rolled at each AI step (which very often), that the AI will try to engulf something it can't eat
+    public const float AI_BAD_ENGULF_CHANCE = 0.25f;
```

Comment:
Even though other people have commented incorrectly the AI constants, could you make this into a proper XML comment?